src/runtime/auto-prompt-manager-test.js

```diff
@@ -443,6 +447,170 @@ describes.realWin('AutoPromptManager', (env) => {
     });
   });
 
+  it('should not set frequency cap local storage if experiment is enabled and content is locked', async () => {
+    setExperiment(win, ExperimentFlags.FREQUENCY_CAPPING_LOCAL_STORAGE, true);
+    autoPromptManager.monetizationPromptWasDisplayedAsSoftPaywall_ = true;
+    sandbox.stub(pageConfig, 'isLocked').returns(true);
+    storageMock
+      .expects('get')
+      .withExactArgs(
+        ImpressionStorageKeys.CONTRIBUTION,
+        /* useLocalStorage */ true
+      )
+      .never();
+    storageMock
+      .expects('set')
+      .withExactArgs(
+        ImpressionStorageKeys.CONTRIBUTION,
+        CURRENT_TIME.toString(),
+        /* useLocalStorage */ true
+      )
+      .never();
+
+    await eventManagerCallback({
+      eventType: AnalyticsEvent.IMPRESSION_CONTRIBUTION_OFFERS,
+      eventOriginator: EventOriginator.UNKNOWN_CLIENT,
+      isFromUserAction: null,
+      additionalParameters: null,
+    });
+  });
+
+  it('should set frequency cap local storage if experiment is enabled and monetization prompts was triggered as a soft paywall', async () => {
```

Comment:
Can we not use the notion of soft paywall in the new flow and just test it between locked and unlocked?

Reply:
Yes, these needed to be updated along with the other change.